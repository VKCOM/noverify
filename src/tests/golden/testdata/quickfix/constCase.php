<?php

function f($a) {
    if ($a === NULL) {
        return FaLse;
    }
    return True;
}

function f1($a) {
    if ($a === null) {
        return false;
    }
    return true;
}

$a = NULL;
$a = null;
