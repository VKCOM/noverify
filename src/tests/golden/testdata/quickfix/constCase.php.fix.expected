<?php

function f($a) {
    if ($a === null) {
        return false;
    }
    return true;
}

function f1($a) {
    if ($a === null) {
        return false;
    }
    return true;
}

$a = null;
$a = null;
